### PR TITLE
Fix duplicate anchors

### PR DESCRIPTION
**Pull request by bedapisl**
Filip: The example in https://github.com/Cognexa/cxflow/pull/222 works for me now, so this code should work too.

### CODE REVIEW

File: emloop/utils/yaml.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def make_simple(data: Any) -> Any:
     :param data: data to be made simple (dict instead of CommentedMap etc.)
     :return: simplified data
     """
-    return yaml.load(yaml.dump(data, Dumper=ruamel.yaml.RoundTripDumper), Loader=ruamel.yaml.Loader)
+    return dict(data)
```
Comment (petrbel) [inline emloop/utils/yaml.py]: if this works (not sure), than the whole `make_simple` is useless and `dict` should be used instead

Comment (FloopCZ) [reply emloop/utils/yaml.py]: Agreed. @bedapisl claims that the original [issue with dict cast](https://bitbucket.org/ruamel/yaml/issues/196/loosing-items-on-cast-to-dict) is fixed.

**Review by FloopCZ — APPROVED** [review]
Thank you @bedapisl !

**Comment by blazekadam** [comment]
Well... this will not convert anything nested right?...

**Comment by bedapisl** [comment]
I don't know what exactly you mean, but the class we are converting is nested in a way (it represents nested yaml configuration) and it looks like it is converted correctly.

**Comment by FloopCZ** [comment]
I actually don't remember why we need the conversion whatsoever. Maybe we can skip the `dict` call entirely.

**Pull request merged**